src/spaceone/inventory/service/collector_service.py

```diff
@@ -82,27 +82,44 @@ def collect(self, params):
             )
             yield error_resource_response.to_primitive()
 
+        # Execute manager (Multithreading)
+        # with concurrent.futures.ThreadPoolExecutor(max_workers=MAX_WORKER) as executor:
+        #     future_executors = []
+        #     for execute_manager in self.execute_managers:
+        #         _manager = self.locator.get_manager(execute_manager)
+        #         future_executors.append(
+        #             executor.submit(_manager.collect_resources, params)
+        #         )
+        #
+        #     for future in concurrent.futures.as_completed(future_executors):
+        #         try:
+        #             for result in future.result():
+        #                 yield result.to_primitive()
+        #         except Exception as e:
+        #             _LOGGER.error(
+        #                 f"[collect] failed to yield result => {e}", exc_info=True
+        #             )
+        #             error_resource_response = self.generate_error_response(
+        #                 e, "", "inventory.Error"
+        #             )
+        #             yield error_resource_response.to_primitive()
+
```

Comment:
The changes can be traced to github's commit, so it would be better to remove the annotation.